### PR TITLE
fix: RDS Certificates PK issue and column names

### PR DESCRIPTION
**Pull request by disq**
PK should be the `arn` (or identifier) and not thumbprint. ~Fixed `_till` columns as well.~

### CODE REVIEW

File: resources/services/rds/certificates.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ func RdsCertificates() *schema.Table {
 		Multiplex:    client.ServiceAccountRegionMultiplexer("rds"),
 		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
 		DeleteFilter: client.DeleteAccountRegionFilter,
-		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "thumbprint"}},
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "arn"}},
```
Comment (bbernays) [inline resources/services/rds/certificates.go:20]: The thing I had been trying to understand was if thumbprint could be replicated accross region, but this seems to have fixed that issue as region is part of arn

Comment (roneli) [reply resources/services/rds/certificates.go:20]: wouldn't account_id be included in arn? we can just use that instead?

**Review by roneli — APPROVED** [review]
LGTM had one extra comment on @bbernays comment

File: resources/services/rds/certificates.go (hunk 1 above)
Comment (disq) [reply resources/services/rds/certificates.go]: correct, fixed

Comment (bbernays) [reply resources/services/rds/certificates.go]: So maybe the better PK choice is `arn` and `thumbprint`?

Comment (bbernays) [reply resources/services/rds/certificates.go]: Actually nevermind, remembered why we need `account_id`

**Pull request merged**